readme.md

```diff
@@ -89,6 +89,7 @@ $ cpy --help
 
   Example
     $ cpy 'src/*.png' dist
+    $ cpy 'src/*.png' '!src/*.ico' dist
```

Comment:
Negation patterns are [deprecated](https://github.com/isaacs/node-glob#comments-and-negation), so we should not propagate their usage. Please remove the exclamation mark.